corehq/apps/sms/filters.py

```diff
@@ -38,6 +38,18 @@ def options(self):
         return options_var
 
 
+class ErrorCodeFilter(BaseMultipleOptionFilter):
+    label = ugettext_noop('Error')
+    default_text = ugettext_noop('Select Error...')
+    slug = 'error_code'
+
+    options = [
+        (code, message.split(".")[0])   # shorten multi-sentence messages
+        for code, message in MessagingEvent.ERROR_MESSAGES.items()
+        if code != MessagingEvent.ERROR_SUBEVENT_ERROR
```

Comment:
why are we ignoring the subevent error code?

Reply:
`ERROR_SUBEVENT_ERROR` is "View details for more information" and indicates there's a more specific error on the subevent. I don't think there's value in filtering for it, since the filter already searches both event and subevent errors.